plugins/ngrok/ngrok.go

```diff
@@ -23,8 +23,7 @@ func ngrokCLI() schema.Executable {
 		),
 		Uses: []schema.CredentialUsage{
 			{
-				Name:        credname.Credentials,
-				Provisioner: ngrokProvisioner(),
```

Comment:
What is the reason and effect of removing this overridden provisioner here?

Reply:
When shell plugins get initialized, all the plugins New() function is called which calls on the ngrokCLI() which calls on Uses and in there, there is a optional provisioner field and currently there is a function that checks whether the version is greater than 3.2.1 and depending if its < or >=, then the type of provisioner is used (either file or using ENV variables).

So by doing any `op plugin` command, it will call `ngrok version`